.ci/azure-pipeline-build.yml

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2019, Henry Schreiner.
+#
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/azure-wheel-helpers for details.
+
+# Template - you must copy (or symlink) this to your .ci folder
+# Packages usually will have a little custom code in here, and
+# can override any of the templates from here, as well
+
+# Builds can always be triggered in the GUI.
+trigger:
+  branches:
+    include:
+      - 'refs/tags/*'
+
+pr: none
+
+# For a master-develop workflow:
+# pr:
+#   - master
+# trigger:
+#   - master
+
+
+
+# You must set the package name here. Optionally change the other variables.
+variables:
+  package_name: numpythia
+  many_linux_base: quay.io/pypa/manylinux1_
+  dev_requirements_file: .ci/azure-wheel-helpers/empty-requirements.txt
+  test_requirements_file: .ci/test-requirements.txt
+  NPY_NUM_BUILD_JOBS: 2
+
+
+jobs:
+
+- job: LinuxSDist
+  pool:
+    vmImage: 'ubuntu-latest'
+  variables:
+    python.architecture: 'none'
+  steps:
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '3.8'
+    - script: |
+        python -m pip install --upgrade pip
+        python -m pip install setuptools numpy
+      displayName: Install sdist deps
+    - script: python setup.py sdist
+      displayName: Prepare sdist
+    - template: azure-wheel-helpers/azure-publish-dist.yml
+
+- job: ManyLinux
+  strategy:
+    matrix:
+      64Bit2010:
+        arch: x86_64
+        plat: manylinux2010_x86_64
+        image: quay.io/pypa/manylinux2010_x86_64
+        python.architecture: x64
+      64Bit:
+        arch: x86_64
+        plat: manylinux1_x86_64
+        image: $(many_linux_base)x86_64
+        python.architecture: x64
+      32Bit:
+        arch: i686
+        plat: manylinux1_i686
+        image: $(many_linux_base)i686
+        python.architecture: x86
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+    - template: azure-wheel-helpers/azure-manylinux-wheels.yml
+    - template: azure-wheel-helpers/azure-publish-dist.yml
+
+- job: macOS
+  variables:
+    python.architecture: 'x64'
+  strategy:
+    matrix:
+      Python27:
+        python.version: '2.7'
+      Python36:
+        python.version: '3.6'
```

Comment:
No 3.5 here?

Reply:
Windows and macOS tend to not be stuck on old versions like Linux users with system Pythons, and macOS doesn't have a pure 64bit 3.5 release I can easily use.